src/fi_tostr.c

```diff
@@ -487,6 +487,14 @@ static void fi_tostr_atomic_op(char *buf, enum fi_op op)
 	}
 }
 
+static void fi_tostr_version(char *buf, uint32_t op)
+{
+	/* PACKAGE_VERSION is defined in config.h by configure */
+	static const char version_str[] = PACKAGE_VERSION;
+	(void)op;
```

Comment:
I think not passing `op` is the better option, so I will fix that in the next PR version.